deploy/v2.0.0-alpha.1/csi-azuredisk-controller.yaml

```diff
@@ -130,7 +130,7 @@ spec:
             - "--v=5"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--metrics-address=0.0.0.0:29604"
-            - "--temp-use-driver-v2"
+            - "--temp-use-driver-v2=true"
```

Comment:
When included in the command line, the value of `--temp-use-driver-v2` is `true`. It is not necessary to explicitly assign it the value `true`. (It doesn't hurt either. :) )